go.mod

```diff
@@ -6,41 +6,44 @@ require (
 	github.com/AndreasBriese/bbloom v0.0.0-20190825152654-46b345b51c96 // indirect
 	github.com/DataDog/zstd v1.4.4 // indirect
 	github.com/Shopify/sarama v1.22.2-0.20190604114437-cd910a683f9f
+	github.com/VividCortex/gohistogram v1.0.0 // indirect
```

Comment:
I don't think there are any changes in this PR that require changing the dependencies, so please revert this. It's preferable to change dependencies (when needed) in a dedicated PR.

Reply:
Unintended for sure.